WordPress/Tests/WhiteSpace/ControlStructureSpacingUnitTest.inc

```diff
@@ -45,3 +45,31 @@ endif;
 if ( false ) :
 else :
 endif;
+
+function($arg){} // Bad
+function ( $arg ) {
```

Comment:
Some of those are already fixed by other sniffs.

`WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterStructureOpen` fixes this:

```php
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use( $var1, $var2 ) {}; // Bad, no space before open parenthesis prohibited.
```
`Squiz.Functions.FunctionDeclarationArgumentSpacing.SpacingBeforeClose` fixes these:

```php
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use ( $var1, $var2) {}; // Bad, no space before closing parenthesis prohibited
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use ( $var1, $var2  ) {}; // Bad, expected exactly one space before closing parenthesis
```
`Squiz.Functions.FunctionDeclarationArgumentSpacing.SpacingAfterOpen` fixes these:

```php
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use ($var1, $var2 ) {}; // Bad, no space after open parenthesis prohibited
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use (  $var1, $var2 ) {}; // Bad, expected exactly one space after opening parenthesis
```

Additionally, `WordPress.WhiteSpace.ControlStructureSpacing.ExtraSpaceAfterCloseParenthesis` fixes this:

```php
$closureWithArgsAndVars = function ( $arg1, $arg2 )  use ( $var1, $var2 ) {}; // Bad, expected exactly one space between closing parenthesis and control structure.
```
`WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterCloseParenthesis` attempts to fix this, however it fails, producing the second line instead:

```php
$closureWithArgsAndVars = function ( $arg1, $arg2 )use ( $var1, $var2 ) {}; // Bad, expected exactly one space between closing parenthesis and control structure.
$closureWithArgsAndVars = function ( $arg1, $arg2 )use ( $var1, $var2 )    {}; // Bad, expected exactly one space between closing parenthesis and control structure.
```

That leaves these:

```php
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use  ( $var1, $var2 ) {}; // Bad, expected exactly one space before opening parenthesis.

$closureWithArgsAndVars = function ( $arg1, $arg2 ) use ( $var1, $var2 ){}; // Bad, space between closing parenthesis and control structure required.
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use ( $var1, $var2 )  {}; // Bad, expected exactly one space between closing parenthesis and control structure.
```